packages/commonwealth/static/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Common App",
+  "short_name": "Common",
+  "description": "Create onchain together - Community forums with onchain rewards, roles and more",
+  "start_url": "/",
+  "display": "standalone",
+  "background_color": "#ffffff",
+  "theme_color": "#2972cc",
+  "icons": [
+    {
+      "src": "/static/brand_assets/512x512.png",
```

Comment:
we should use new common logo. We already have `img/branding` directory, where new files live.

Reply:
Unfortunately this didn't have the new icons we needed, but I was able to add them in.